src/integrationTest/java/uk/gov/hmcts/darts/dailylist/service/DailyListProcessorTest.java

```diff
@@ -98,6 +99,43 @@ void closeHibernateSession() {
     }
 
 
+    @Test
+    void dailyListProcessor_missingForename_shouldStillCreateUsers() throws IOException {
```

Comment:
I was expecting to see a test which verified that a slightly different duplicate name was not inserted?